pkg/extender/extender.go

```diff
@@ -123,6 +126,20 @@ func (e *Extender) processFilterRequest(w http.ResponseWriter, req *http.Request
 	}
 
 	pod := args.Pod
+	for _, vol := range pod.Spec.Volumes {
+		// if any of pvc has restore annotation skip scheduling pod
+		pvc, err := k8s.Instance().GetPersistentVolumeClaim(vol.PersistentVolumeClaim.ClaimName, pod.Namespace)
+		if err != nil {
+			msg := "Unable to find PVC claim"
+			storklog.PodLog(pod).Warnf(msg)
```

Comment:
Need to respond to the request with an error with `http.Error(w, ...)` and return

Reply:
Still missing `http.Error(w, ...)` here?

Reply:
if pod does not have valid volumes claim should we stop scheduling or just warn in log and go ahead